ravenframework/Samplers/MonteCarlo.py

```diff
@@ -131,6 +127,10 @@ def localGenerateInput(self, model, myInput):
           lower = distData['xMin']
           upper = distData['xMax']
           rvsnum = lower + (upper - lower) * randomUtils.random()
+          # TODO (wangc): I think the calculation for epsilon need to be updated as following
+          """
+            epsilon = (upper-lower)/(self.limit+1) * 0.5
+          """
```

Comment:
I think docstrings are discouraged here, can we place it in comment block instead such as:
```python
# TODO (wangc): etc etc
#
#   epsilon = etc etc
#

```

Reply:
changed.